### DISCUSSION TITLE
how i create binaries file from prisma/engine to PRISMA_ENGINES_MIRROR in folder all_commits

### DISCUSSION THREAD

**Discussion by ofekBerko1** [prisma/prisma#20963] [Q&A]
### Question

i want to creacte custom server to prisma engines. for this, i need to create binaries files in all_commits folder. how i do this?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @Ofek1502 👋 

Can you please follow the instructions in this repository to create Prisma engines?

- https://github.com/prisma/prisma-engines#building-prisma-engines

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
